doc/source/whatsnew/v1.2.0.rst

```diff
@@ -286,6 +286,8 @@ Other enhancements
 - Added methods :meth:`IntegerArray.prod`, :meth:`IntegerArray.min`, and :meth:`IntegerArray.max` (:issue:`33790`)
 - Calling a NumPy ufunc on a ``DataFrame`` with extension types now preserves the extension types when possible (:issue:`23743`)
 - Calling a binary-input NumPy ufunc on multiple ``DataFrame`` objects now aligns, matching the behavior of binary operations and ufuncs on ``Series`` (:issue:`23743`).
+  This change has been reverted in pandas 1.2.1, and the behaviour to not align DataFrames
```

Comment:
you may want to make this a `::note` in the docs itself / or put it uptop as its not obvious this changed unless you look in detail. (this is just to call out the visibility). another alternative is to have another sub-section 'Changes in version for this series e.g. in the 1.2.x). just throwing out some ideas here.

Reply:
Yes, I was thinking how I could give it more visibility (see also https://github.com/pandas-dev/pandas/pull/39239#issuecomment-762921620), but because we didn't have subsections in the "Deprecations" section yet, didn't add one there. But maybe still worth to add one anyway.

Reply:
@jreback I added another note in the "Deprecations" section (using a semi-header in bold font, which is how we also did it in the v1.0.0.rst docs for a note that needed to stand out a bit from the other listed items). Can you take a look at it?

Reply:
ahh ok, sure that's reasonable